langserver/langserver_test.go

```diff
@@ -73,6 +75,7 @@ func TestServer(t *testing.T) {
 				"B":           []string{"/src/test/pkg/b.go:function:pkg.B:0:16"},
 				"is:exported": []string{"/src/test/pkg/a.go:function:pkg.A:0:16", "/src/test/pkg/b.go:function:pkg.B:0:16"},
 			},
+			wantWorkspaceReferences: []string{},
```

Comment:
No, we're explicitly saying "no workspace references". Omitting the field would say "do not call workspace references at all"